addon/components/offering-form.hbs

```diff
@@ -222,6 +222,18 @@
           />
         </div>
       {{/if}}
+      {{#unless @smallGroupMode}}
+        <div class="learners item">
+          <label>
+            {{t "general.learners"}}:
+          </label>
+            <LearnerSelectionManager
+              @learners={{this.learners}}
+              @add={{fn this.addLearner}}
```

Comment:
`fn` is unneeded here since `addLearners` is a function already and there are no additional arguments to be added at this point.

Reply:
fixed.